### PR TITLE
Feature 1 Allow FE to talk with BE

### PR DESCRIPTION
**Pull request by zenajun**
I changed the cors origin to  `localhost:3000`

**Question**: After we deploy our FE to Netlify, can change  http://${process.env.PROJECT_NAME}-frontend.bridgeschoolapp.io to the Netlify URL?

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -16,7 +16,7 @@ const { channelRouter } = require("./routes/channels/channels.router");
 const app = express();
 
 // The port the express app will listen on
-const port = process.env.PORT || 8081;
+const port = process.env.PORT;
```
Comment (MustafaHaddara) [inline src/index.js]: this means `PORT` **must** be set on in your environment variables...is that intentional?

File: src/index.js [hunk 2]
```diff
@@ -25,7 +25,8 @@ logger.info("🤖 Initializing middleware");
 app.use(morgan("tiny", { stream: logger.stream }));
 app.use(
   cors({
-    origin: `http://${process.env.PROJECT_NAME}-frontend.bridgeschoolapp.io`
+    origin: `http://localhost:3000`
+    // origin: `http://${process.env.PROJECT_NAME}-frontend.bridgeschoolapp.io`
```
Comment (MustafaHaddara) [inline src/index.js]: I took a look at [the `cors` docs](https://www.npmjs.com/package/cors#configuration-options), apparently `origin` can be an array! So you can do

```suggestion
origin: [`http://localhost:3000`,  `blah-blah.netlfiy.com`, `http://${process.env.PROJECT_NAME}-frontend.bridgeschoolapp.io`]
```

and they'll all be allowed!

File: src/index.js (hunk 1 above)
Comment (zenajun) [reply src/index.js]: Yes! We have the variable set in the .env 👍

File: src/index.js (hunk 2 above)
Comment (zenajun) [reply src/index.js]: 🤸‍♀️ Sweet, thanks!

**Pull request merged**